libjolie/src/jolie/lang/parse/util/impl/ProgramInspectorCreatorVisitor.java

```diff
@@ -367,4 +295,9 @@ public void visit( InlineTreeExpressionNode n ) {}
 	public void visit( VoidExpressionNode n ) {}
 	@Override
 	public void visit( ProvideUntilStatement n ) {}
+
+	@Override
+	public void visit(TypeChoiceDefinition n) {
```

Comment:
Shouldn't it do something? We are doing something (registration) for the other types.